config/default.json

```diff
@@ -0,0 +1,12 @@
+{
+	"curator": {
+		"languages": {
+			"rb": "Ruby",
+			"cs": "C#",
+			"js": "JavaScript"
+		},
+		"commands": [
+			{"init": "initialize cwd with _templates folder, stream-specific subfolders, and template md files"}
```

Comment:
commands being an array is kinda weird actually. would be more parseable if it looked like languages

whatcha think? @amelialaundy